### ISSUE TITLE
Unknown method 'platform'

### ISSUE THREAD

**Issue by Robotex** [fastlane/fastlane-plugin-firebase_app_distribution#154]
Fastlane throws this error
ERROR [2020-09-21 04:00:36.66]: Unknown method 'platform'
ERROR [2020-09-21 04:00:36.66]: To call another action from an action use `other_action.platform` instead

Call stack:

```
FastlaneCore::Interface::FastlaneError: [!] To call another action from an action use `other_action.platform` instead
  /Users/robotex/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/fastlane-2.160.0/fastlane_core/lib/fastlane_core/ui/interface.rb:141:in `user_error!'
  /Users/robotex/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/fastlane-2.160.0/fastlane_core/lib/fastlane_core/ui/ui.rb:17:in `method_missing'
  /Users/robotex/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/fastlane-2.160.0/fastlane/lib/fastlane/action.rb:134:in `method_missing'
  /Users/robotex/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/bundler/gems/fastlane-plugin-firebase_app_distribution-6c14d70ddb20/lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb:26:in `run'
```

Could it be because the variable `platform` here
https://github.com/fastlane/fastlane-plugin-firebase_app_distribution/blob/44d604221c7b92f05334d47fced149f47fb9d624/lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb#L26

is being used before being declared here?
https://github.com/fastlane/fastlane-plugin-firebase_app_distribution/blob/44d604221c7b92f05334d47fced149f47fb9d624/lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb#L36

Version: 0.2.2.pre.1 on master branch

**Comment by lfkellogg** [comment]
Good catch, thanks! This should fix it: https://github.com/fastlane/fastlane-plugin-firebase_app_distribution/pull/155

**Comment by lfkellogg** [comment]
@Robotex The fix has been released in `0.2.2.pre.2`. Could you please update and verify that it fixes the issue for you?

**Comment by tagboola** [comment]
@Robotex  the fix has been released in `0.2.2`. Going to close this issue for now.
